test/fulfillmentSpec.js

```diff
@@ -348,6 +349,12 @@ describe('PUT /fulfillment', function () {
     })
 
   it('should not double spend when transfer is executed multiple times', function * () {
+    // Pre-fund hold account to allow double spending of source transfer
+    // This could happen if, for ex, 2 txn are escrowed, then first is executed twice.
```

Comment:
@justmoon @emschwartz I'm not very familiar with the code, does anyone want to look at this?